CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ Breaking changes
   When creating a new bucket instance a model instance must be provided instead
   of `orm.SimpleObj`.
 
+=======
```

Comment:
`======` is not part of markdown and [it renders as several `=` signs](https://github.com/iov-one/weave/blob/997c0daa9b003aa0ec0531b5b3ca6a282cfea216/CHANGELOG.md) :nail_care: